### PR TITLE
FAQ: More about preservation of changes and the MCO

### PR DESCRIPTION
**Pull request by cgwalters**
Package layering came up in a Slack conversation.

### CODE REVIEW

**Comment by kikisdeliveryservice** [comment]
stop it bot!

/skip

**Comment by ashcrow** [comment]
Overriding bot as this is clarifying documentation.

File: docs/FAQ.md [hunk 1]
```diff
@@ -44,7 +44,11 @@ Usually, no.  Today, the MCO does not try to claim "exclusive" ownership over ev
 
 If for example you write a daemonset that writes a custom systemd unit into e.g. `/etc/systemd/system`, or do so manually via `ssh`/`oc debug node` - OS upgrades will preserve that change (via libostree), and the MCO will not revert it.  The MCO/MCD only changes files included in `MachineConfigs`, there is no code to look for "unknown" files.
 
-Another case today is that the SDN operator will extract some binaries from its container image and drop them in `/opt`.
+Another case today is that the SDN operator will extract some binaries from its container image and drop them in `/opt` (which is really `/var/opt`).
+
+Stated more generally, on an OSTree managed system, all content in `/etc` and `/var` is [preserved by default across upgrades](https://ostree.readthedocs.io/en/latest/manual/adapting-existing/).
```
Comment (wking) [inline docs/FAQ.md]: Are we concerned about this floating master link?  I'd have expected something pinned to a specific OSTree release.

**Pull request merged**